db_processor.bal

```diff
@@ -217,8 +217,8 @@ function getAidPackages(string? status) returns AidPackage[]|error {
 function addAidPackage(AidPackage aidPackage) returns int|error {
     int packageId = -1;
     int currentTime = getEpoch();
-    sql:ExecutionResult result = check dbClient->execute(`INSERT INTO AID_PACKAGE(NAME, DESCRIPTION, STATUS, DATETIME)
-                                        VALUES (${aidPackage.name}, ${aidPackage.description}, ${aidPackage.status}, FROM_UNIXTIME(${currentTime}));`);
+    sql:ExecutionResult result = check dbClient->execute(`INSERT INTO AID_PACKAGE(NAME, DESCRIPTION, STATUS, DATETIME, CREATEDBY)
+                                        VALUES (${aidPackage.name}, ${aidPackage.description}, ${aidPackage.status}, FROM_UNIXTIME(${currentTime}), ${aidPackage?.createdBy});`);
```

Comment:
What will happen when FE doesn't send a createdBy?

Reply:
I think this is mandatory. Hence, they have to send it.